client/stylesheets/_toCat.scss

```diff
@@ -5464,12 +5479,16 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 0;
 }
 
+.profileUserAmount ul li span {
+    display: inline-block !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces